src/AmperkaTDS.cpp

```diff
@@ -0,0 +1,45 @@
+#include <AmperkaTDS.h>
+
+AmperkaTDS::AmperkaTDS(uint8_t pinS) : _pinS(pinS) {}
+
+void AmperkaTDS::begin() {
+    pinMode(this->_pinS, INPUT);
```

Comment:
Why "this->" used? Within the class implementation, you can simply use the names of variables and methods. It makes sense to use "this->" only if the class fields have the same names as the method parameters.